routing/router.go

```diff
@@ -885,39 +910,27 @@ func (r *ChannelRouter) pruneZombieChans() error {
 			return nil
 		}
 
-		// If either edge hasn't been updated for a period of
-		// chanExpiry, then we'll mark the channel itself as eligible
-		// for graph pruning.
-		e1Zombie := e1 == nil || time.Since(e1.LastUpdate) >= chanExpiry
-		e2Zombie := e2 == nil || time.Since(e2.LastUpdate) >= chanExpiry
-
-		if e1Zombie {
+		if e1 == nil || time.Since(e1.LastUpdate) >= chanExpiry {
 			log.Tracef("Node1 pubkey=%x of chan_id=%v is zombie",
 				info.NodeKey1Bytes, info.ChannelID)
 		}
-		if e2Zombie {
+		if e2 == nil || time.Since(e2.LastUpdate) >= chanExpiry {
```

Comment:
If we want to fully consolidate this code block, then we can have an internal version of `IsZombieChannel` that returns a bool for `e1` and `e2`. Then we check the conjunction below.